### ISSUE TITLE
Carla 2.4.3 not search .dll

### ISSUE THREAD

**Issue by A350** [falkTX/Carla#1785]
The .dll do not appear in the search in Carla 2.4.3 in debian , but they work if you load them... is it a bug ?

**Comment by falkTX** [comment]
why using 2.4.3?? latest is 2.5.5, use that please.

**Comment by A350** [comment]
I'm still on debian 10, I think it's the last stable for Buster, isn't it ?

**Comment by falkTX** [comment]
carla is not on the debian repos, so for sure you got it from somewhere else.

**Comment by A350** [comment]
yes of course, in kxstudio-repos

**Comment by falkTX** [comment]
right, there are no more updates for debian 10, the minimum is 11.

but debian 10 is now old-old-stable, which is really not supported anymore.
build carla yourself if you want the new version, but you are on your own for getting the bridges to work.
the install file on the repo has some instructions, good luck!

**Comment by A350** [comment]
yes, i known.. Thanks for the support !
